src/main/java/seedu/canoe/logic/commands/AddStudentToTrainingCommand.java

```diff
@@ -82,6 +83,11 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Training trainingToEdit = lastShownList.get(index.getZeroBased());
+        /* CONDITION TO CHECK IF THE CURRENT DATE TIME IS ALREADY AFTER TRAINING'S STARTING DATE TIME
```

Comment:
is this part removed?

Reply:
nope, it's left here since `AddStudentToTrainingCommandTest` has not been updated to factor in this check